Resources/Prototypes/_White/Entities/Markers/Spawners/mobs.yml

```
@@ -0,0 +1,13 @@
+- type: entity
+  parent: MarkerBase
+  id: SpawnMobGondola
+  name: Gondola spawner
+  components:
+  - type: Sprite
+    layers:
+      - state: green
```

yamllint

[warning] wrong indentation: expected 4 but found 6

(indentation)